src/relayserver/ServerWinstonLogger.ts

```diff
@@ -0,0 +1,55 @@
+import winston, { Logger, transport } from 'winston'
+import { ConsoleTransportOptions, HttpTransportOptions } from 'winston/lib/winston/transports'
+import { URL } from 'url'
+
+import { gsnRuntimeVersion } from '../common/Version'
+
+import { NpmLogLevel } from '../relayclient/types/Aliases'
+
+const service = 'gsn-server'
+const filename = 'combined.log'
+
+const format = winston.format.combine(
+  winston.format.uncolorize(),
+  winston.format.timestamp(),
+  winston.format.simple()
+)
+
+const consoleOptions: ConsoleTransportOptions = {
+  format: winston.format.combine(
+    winston.format.cli()
+  )
+}
+
+export function createLogger (level: NpmLogLevel, loggerUrl: string, userId: string): Logger {
+  const transports: transport[] = [
+    new winston.transports.Console(consoleOptions),
+    new winston.transports.File({ format, filename })
```

Comment:
don't force both file and console logs. file should be logged only if filename given.

Reply:
why do we need file logger? if we add one, we also need to add facility to rotate the log file, etc. (otherwise, it can grow indefinitely )
I think we can remove file logger for now.

Reply:
Removed file log until further discussion.